addons/website_slides/views/slide_slide_views.xml

```diff
@@ -203,67 +203,52 @@
-                                                <i class="fa fa-file-image-o me-2" aria-label="Infographic" role="img" title="Infographic"/>
-                                            </t>
-                                            <t t-elif="record.slide_category.raw_value == 'article'">
-                                                <i class="fa fa-file-code-o me-2" aria-label="article" role="img" title="Article"/>
-                                            </t>
-                                            <t t-elif="record.slide_category.raw_value == 'video'">
-                                                <i class="fa fa-file-video-o me-2" aria-label="Video" role="img" title="Video"/>
-                                            </t>
-                                            <t t-elif="record.slide_category.raw_value == 'quiz'">
-                                                <i class="fa fa-flag me-2" aria-label="Quiz" role="img" title="Quiz"/>
-                                            </t>
-                                            <t t-else=""><i class="fa fa-file-pdf-o me-2" aria-label="Document" role="img" title="Document"/></t>
-                                            <field name="slide_category"/>
-                                        </span>
-                                        <span>
-                                            <i class="fa fa-clock-o me-2" aria-label="Duration" role="img" title="Duration"/><field name="completion_time" widget="float_time"/>
-                                        </span>
-                                        <span>
-                                            <i class="fa fa-question me-2" aria-label="Number of Questions" role="img" title="Number of Questions"/><field name="questions_count"/>
-                                        </span>
-                                        <span>
-                                            <i class="fa fa-eye me-2" aria-label="Views" role="img" title="Views"/><field name="total_views"/>
-                                        </span>
-                                        <field name="user_id" widget="many2one_avatar_user"/>
+                        <t t-name="kanban-card" class="flex-row">
+                            <aside class="o_kanban_aside_full">
+                                <t t-if="record.image_128.raw_value">
+                                    <div class="o_kanban_image_fill position-relative w-100">
+                                        <field name="image_128" t-att-alt="record.id.value" class="h-100" widget="image" options="{'img_class': 'object-fit-cover'}"/>
+                                        <field name="channel_id" class="position-absolute bottom-0 end-0 bg-light" widget="image" options="{'preview_image': 'image_128', 'img_class': 'object-fit-contain'}" style="box-shadow: -1px -1px 0 1px white; width: 40%;"/>
```

Comment:
I would avoid inline style and define it in scss instead. Where does this style come from by the way?

Reply:
i have added new class `o_inner_image`, the style comes from [o_kanban_image_inner_pic](https://github.com/odoo/odoo/blob/0b8b14c5ae5ff06ce8914a94b602a7fac2c2655b/addons/web/static/src/views/kanban/kanban_record_legacy.scss#L137) here